js/ToDoList.js

```diff
@@ -0,0 +1,52 @@
+import { Todoitem } from "./TodoItem .js";
+
+class ToDoList {
+  description_task = document.getElementById("task-input");
+
```

Comment:
please remove this var, as it's not needed, as we can call `document.get` from anywhere, so you can use it from the app itself